doc/api/deprecations.md

```diff
@@ -2336,6 +2335,19 @@ Type: Runtime
 Setting the TLS ServerName to an IP address is not permitted by
 [RFC 6066][]. This will be ignored in a future version.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: using REPLServer#rli
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+This property is just a reference to the instance itself.
```

Comment:
Super-micro-ignoreable-nitpick: 

Maybe one of these three instead of "just"?

Option 1: Omit it.

```suggestion
This property is a reference to the instance itself.
```

Reply:
Option 2: "only"

```suggestion
This property is only a reference to the instance itself.
```

Reply:
Option 3: "unnecessary" or "unneeded"?

```suggestion
Option 2: "only"

```suggestion
This property is an unneeded reference to the instance itself.
```